ironfish/src/telemetry/telemetry.ts

```diff
@@ -354,6 +354,37 @@ export class Telemetry {
     })
   }
 
+  submitNewTransactionCreated(transaction: Transaction, seenAt: Date): void {
+    const hash = transaction.hash()
+
+    if (!this.shouldSubmitTransaction(hash)) {
+      return
+    }
+
+    this.submit({
+      measurement: 'transaction_created',
+      timestamp: seenAt,
+      tags: [
+        {
+          name: 'hash',
+          value: hash.toString('hex'),
+        },
+      ],
+      fields: [
+        {
+          name: 'notes',
+          type: 'integer',
+          value: transaction.notesLength(),
+        },
+        {
+          name: 'spends',
+          type: 'integer',
+          value: transaction.spendsLength(),
```

Comment:
If we submit these, should we remove these points for `transaction_propagation`?

Reply:
Yea I think we can. I will do that